sdk/lib/rtl/acl.c

```diff
@@ -847,21 +847,21 @@ RtlValidAcl(IN PACL Acl)
         if ((Acl->AclRevision < MIN_ACL_REVISION) ||
             (Acl->AclRevision > MAX_ACL_REVISION))
         {
-            DPRINT1("Invalid ACL revision\n");
+            DPRINT1("Invalid ACL revision: %hhu\n", Acl->AclRevision);
```

Comment:
`%hhu` (or even `%hu`) is not a thing, integers get promoted. Just %u will be fine.

Reply:
As you prefer. Done.